src/tests/unit/components/category-item-card/CategoryItemCard.spec.jsx

```diff
@@ -0,0 +1,36 @@
+import { render, screen, waitFor } from '@testing-library/react'
+import CategoryItemCard from '~/components/category-item-card/CategoryItemCard'
+
+const mockProps = {
+  link: 'https://example.com',
+  image: 'https://example.com/image.jpg',
+  title: 'Test Title',
+  description: 'Test Description'
+}
+
+describe('CategoryItemCard', () => {
+  beforeEach(async () => {
+    await waitFor(() => {
```

Comment:
is any specific reason for using 'await waitFor here'?

Reply:
I'm sorry! I changed it!